tests/phpunit/tests/blocks/render.php

```diff
@@ -233,6 +233,10 @@ public function test_do_block_output( $html_filename, $server_html_filename ) {
 		$normalized_html = preg_replace( '/wp-block-gallery-\d+/', 'wp-block-gallery-1', $normalized_html );
 		$expected_html   = self::strip_r( file_get_contents( $server_html_path ) );
 
+		// Convert HTML to be white space insensitive.
+		$normalized_html = preg_replace( '/(\s+$)/m', '', $normalized_html );
+		$expected_html   = preg_replace( '/(\s+$)/m', '', $expected_html );
+
```

Comment:
This change strips whitespace from the end of lines in both the normalized and expected HTML.

When rendering the blocks, Gutenberg often adds a white space following any dynamic attributes, eg `data-id="1234" ` which is correct but if it comes in at the end of the line, can be missed in the expected version.

As HTML treats one or more instances of white space identically, it doesn't have an effect on the behavior in browsers.